maps-utils-ktx/src/main/java/com/google/maps/android/ktx/utils/geojson/GeoJson.kt

```diff
@@ -45,3 +47,26 @@ inline fun geoJsonLayer(
     polylineManager,
     groundOverlayManager
 )
+
+/**
+ * Alias for the [GeoJsonLayer] constructor that provides Kotlin named parameters and default
+ * values.
+ */
+// FIXME - Why isn't this working with named parameters in demo app?
```

Comment:
@arriolac I'd like to used named parameters in `MainActivity` for the GeoJson layer, which needs another alias for a different constructor that takes in the resource ID and context. I've tried to add this below, but it doesn't seem to work when changing to named parameters in `MainActivity`. Anything obvious that I'm missing?